src/libData/AccountData/Account.cpp

```diff
@@ -135,7 +135,13 @@ void AccountBase::SetCodeHash(const dev::h256& codeHash) {
 
 const dev::h256& AccountBase::GetCodeHash() const { return m_codeHash; }
 
-bool AccountBase::isContract() const { return m_codeHash != dev::h256(); }
+bool Account::isContract() const {
+  return (m_codeHash != dev::h256() && !m_is_library);
```

Comment:
A small suggestion: better to put the "cheap" condition first:
`return  !m_is_library && m_codeHash != dev::h256()`

Reply:
done